tools/post-release.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
```

Comment:
While I think that we need to break down `devtool` into multiple pieces, is there a reason for declaring the functions below in a different file? It seems like this file is only called from devtool once.

If we decide to break devtool down, we need to do a nicer modularization that shares things like `say_err` or `die`.